From 48c9750798e2e06baf61f598ba4715ad402b3493 Mon Sep 17 00:00:00 2001
From: Kevin Haggerty <haggertk@lineageos.org>
Date: Sun, 3 Nov 2019 01:36:15 +0200
Subject: [PATCH] sepolicy: Label sysfs_net

 * avc: denied { write } for name=mtu dev=sysfs ino=24282
   scontext=u:r:netd:s0 tcontext=u:object_r:sysfs:s0 tclass=file permissive=1

   avc: denied { open } for name=mtu dev=sysfs ino=24282
   scontext=u:r:netd:s0 tcontext=u:object_r:sysfs:s0 tclass=file permissive=1

Change-Id: I6f40b8bdac2537b7000c02af6fac8277acb2a718
---
 sepolicy/file_contexts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 4540ce06..7cb352a2 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -159,6 +159,9 @@
 
 /sys/devices/msm_sdcc.1/mmc_host/mmc1/mmc1:0001/block/mmcblk0/stat      u:object_r:sysfs_disk_stat:s0
 
+#sysfs - net
+/sys/devices/virtual/net(/.*)?   u:object_r:sysfs_net:s0
+
 # Sysfs files used by qmuxd
 /sys/devices/virtual/smdpkt/smdcntl([0-9])+/open_timeout u:object_r:sysfs_smdcntl_open_timeout:s0
 
